Dockerfile

```diff
@@ -5,16 +5,16 @@ ENV LANG C.UTF-8
 WORKDIR /srv
 
 # System dependencies
-RUN apt-get update && apt-get install --yes python3-pip
+RUN apt-get update && apt-get install --no-install-recommends --yes python3 python3-setuptools python3-pip
 
 # Import code, install code dependencies
 ADD . .
-RUN pip3 install -r requirements.txt
+RUN python3 -m pip install --no-cache-dir -r requirements.txt
 
 # Set git commit ID
 ARG COMMIT_ID
-RUN test -n "${COMMIT_ID}"
 RUN echo "${COMMIT_ID}" > version-info.txt
+ENV COMMIT_ID "${COMMIT_ID}"
```

Comment:
No `TALISKER_REVISION_ID`?

Reply:
Not sure, we need to look at this, since different projects seem to use different namings. In the jenkins for `canonica.com` job we need `COMMIT_ID`.